EIPS/eip-7085.md

```diff
@@ -0,0 +1,116 @@
+
+Our proposed EIP builds upon the existing Ethereum ecosystem and extends the standard for NFTs.
+
+## Specification
+
+This EIP proposes the addition of two new functions to the [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) standard: `setRelationship` and `setAttribute`.
+
+### `setRelationship`
+
+The `setRelationship` function establishes a referring relationship between two NFTs. It takes the following parameters:
+
+```solidity
+function setRelationship(uint256 _originalID, uint256 _derivativeID, uint256 _attribute) external;
+```
+
+- `_originalID`: the ID of the original NFT
+- `_derivativeID`: the ID of the derivative NFT that refers to the original NFT
+- `_attribute`: the quantifiable attribute for this relationship, which defaults to 1 if not specified
+
+When called, this function establishes a referring relationship between the two NFTs.
+
+### `setAttribute`
+
+The `setAttribute` function sets a quantifiable attribute for an NFT. It takes the following parameters:
+
+```solidity
+function setAttribute(uint256 _id, string calldata _name, uint256 _value) external;
+```
+
+- `_id`: the ID of the NFT
+- `_name`: the name of the attribute to be set
+- `_value`: the value of the attribute to be set
+
+When called, this function sets a quantifiable attribute for the NFT.
+
+### Backward Compatibility
+
+This EIP is designed to be fully backward-compatible with existing [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md)  contracts and tokens. Existing NFT contracts and tokens will continue to function as they did before, and the new `setRelationship` and `setAttribute` functions will only be available to contracts that explicitly implement this EIP.
+
+### Gas Considerations
+
+To mitigate potential issues with excessive complexity and gas cost, the `setRelationship` function only allows for one referring relationship to be established between two NFTs, and the `setAttribute` function only allows for one attribute to be set at a time.
+
+## Rationale
+
+The [ERC-721](./eip-721.md) and [ERC-1155](./eip-1155.md) standard has been widely adopted for the creation of non-fungible tokens (NFTs) and has proven to be a versatile and flexible standard. However, there is currently no way to establish relationships between NFTs or to set quantifiable attributes for those relationships. This limits the potential use cases for NFTs, as it makes it difficult to track ownership history, establish provenance, and create interactive games and experiences.
+
+The motivation behind proposing this EIP is to provide a way for NFT creators and users to establish relationships between NFTs and set quantifiable attributes for those relationships. This will enable more complex and interesting use cases for NFTs, such as tracking ownership history, establishing provenance, and creating interactive games and experiences.
```

Comment:
These bits belong in your Motivation (and I think are adequately covered there already.)

```suggestion
```